tests/test_phase8_a1_domain_primitives_entitlement.py

```
@@ -0,0 +1,729 @@
+    event = EntitlementEvent(
+        event_id=event_id,
+        entitlement_id=entitlement_id,
+        target_seat_id=target_seat_id,
+        class_id=class_id,
+        entitlement_type=entitlement_type,
+        event_type=event_type,
+        acquisition_type=acquisition_type,
+        product_id=product_id,
+        correlation_id=correlation_id,
+        actor_seat_id=actor_seat_id,
+        timestamp=timestamp or datetime.utcnow(),
```

Ruff (0.16.0)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)